website/typedoc.json

```diff
@@ -0,0 +1,38 @@
+{
```

Comment:
each package should generate its own docs

then website declare dependencies to packages

and import package docs through node_modules

we will configure turbo in order to build package docs when whe build website